tests/input/test_inputconfigurator.py

```diff
@@ -191,3 +191,20 @@ def test_export_schema_to_yaml(self, tmp_path):
         loaded_schema_file = yaml.safe_load(file_content)
         assert loaded_schema_file == schema
         assert yaml_str_2 == file_content
```

Comment:
Let's use descriptive variable names. Is there a `yaml_str_1` somewhere in this file?